.pre-commit-hooks.yaml

```diff
@@ -109,6 +109,30 @@
     fundamentally requires scanning the entire repository and hence can't be
     performed incrementally.
 
+- id: megalinter-incremental
+  name: Run MegaLinter (skipping linters that run in project mode)
+  entry: >
+    npx -- mega-linter-runner@v6.7.1
+    --containername megalinter-$(basename "$PWD")
+    --remove-container
+    --fix
+    --env CLEAR_REPORT_FOLDER=true
+    --env LOG_LEVEL=warning
+    --filesonly
+  language: system
+  description: >
```

Comment:
We want to specify that this runs pre-commit here since that is true organization-wide for us.